libs/shared/console-shared/src/lib/flow-create-port/ui/crud-modal/crud-modal.tsx

```diff
@@ -232,6 +258,82 @@ export function CrudModal({
             />
             <p className="mb-5 ml-3 text-xs text-neutral-350">{`Port Name allows to customize the subdomain assigned to reach the application port from the internet. Default value is p<port_number>`}</p>
           </div>
+
+          <div className="space-y-4">
+            <hr />
+            <div className="flex items-center justify-between">
+              <div className="flex items-center gap-2 py-2">
+                <Controller
+                  name="rewrite_public_path"
+                  control={control}
+                  render={({ field }) => (
+                    <>
+                      <Checkbox
+                        name={field.name}
+                        id={field.name}
+                        checked={field.value}
+                        onCheckedChange={(checked) => {
+                          if (!checked) {
+                            setValue('public_path', '')
+                            setValue('public_path_rewrite', '')
+                          }
+
+                          field.onChange(checked)
+                        }}
+                      />
+                      <label htmlFor="rewrite_public_path" className="text-sm font-medium text-neutral-400">
+                        Rewrite public URL
+                      </label>
+                    </>
+                  )}
+                />
+              </div>
+              <Link
+                to="https://kubernetes.github.io/ingress-nginx/examples/rewrite/#rewrite-target"
+                color="brand"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                Documentation
+                <Icon iconName="file-lines" iconStyle="regular" />
```

Comment:
Are we sure about this link we don't use it anywhere?
You should use the component ` <ExternalLink />`